src/main/java/bio/terra/janitor/service/cleanup/MetricsHelper.java

```diff
@@ -142,4 +166,14 @@ public static void recordResourceKindCount(ResourceKindCount count) {
             .build();
     STATS_RECORDER.newMeasureMap().put(TRACKED_RESOURCE_COUNT, count.count()).record(tctx);
   }
+
+  /** Increment the count of the cleanup flights that were already submitted to Stairway but on recovery were still in the {@link bio.terra.janitor.db.CleanupFlightState#INITIATING} state.*/
```

Comment:
Feel this line too long. ditto for the javadoc below

Reply:
done